README.md

```diff
@@ -128,3 +187,108 @@ Beanstalk to retry it later, you should map SilentFailingListener to the message
         'user.updated' => ZfrEbWorker\Listener\SilentFailingListener::class,
     ]
 ```
+
+### How to use periodic tasks?
+
+Elastic Beanstalk also supports periodic tasks through the usage of `cron.yaml` file. However, this is actually easier as you can
+specify a different URL on a task-basis. Therefore, you can dispatch to the URL of your choice and immediately be re-routed to the
+correct middleware.
+
+## CLI commands
+
+Starting from version 3.3, ZfrEbWorker comes with Symfony CLI commands that allows:
+ 
+* to easily push messages into a queue that respect the ZfrEbWorker format.
+* to emulate the usage of native Elastic Beanstalk worker to fetch messages and execute them.
+
+> This local worker is only meant to be used in development. In production, you should use the native Elastic Beanstalk worker, which
+is much faster (retrieves up to 10 messages in one SQS call) and is built-in into the Elastic Beanstalk AMI (it is monitored...).
+
+### Setup
+
+Before using those CLI commands, there are some things you need to setup, as described in following sections.
+
+#### Add the dependencies
+
+Make sure that you add those two dependencies in your project (typically, in the `require-dev` section of your `composer.json` file):
+
+```json
+{
+  "require-dev": {
+    "symfony/console": "^3.0",
+    "guzzlehttp/guzzle": "^6.0"
+  }
+}
+```
+
+#### Adding a console entry point
+
+ZfrEbWorker adds the `WorkerCommand` and `PublisherCommand` Symfony CLI command into the `console` top-key of the config. If you are using this library
+with a framework that already uses Symfony CLI, just add the `ZfrEbWorker\Cli\WorkerCommand` and/or `ZfrEbWorker\Cli\PublisherCommand` commands.
+
+If you are using Zend\Expressive, here is a sample file (call it `console.php` for instance) you can add into the `public` folder, 
+alongside your `index.php` file:
+
+```php
+require_once 'vendor/autoload.php';
+
+use Symfony\Component\Console\Application;
+use Zend\ServiceManager\ServiceManager;
+use ZfrEbWorker\ModuleConfig;
+
+$config = (new ModuleConfig())->__invoke();
```

Comment:
Won't work because it needs application config as well (in order to build the AwsSdk)